components/opponent/wikis/smash/opponent_custom.lua

```diff
@@ -34,7 +34,7 @@ function CustomOpponent.readOpponentArgs(args)
 	local game = args.game or Variables.varDefault('tournament_game') or Info.defaultGame
 	opponent.players[1].game = game
 
-	local CharacterStandardizationData = mw.loadData('Module:CharacterStandardization/' .. game)
+	local CharacterStandardizationData = mw.loadData('Module:CharacterStandardization')
```

Comment:
if not game depending move up to the other imports as
`local CharacterStandardizationData = Lua.import('module:CharacterStandardization', {loadData = true})`